kpi/models/asset.py

```diff
@@ -596,23 +596,28 @@ class Meta:
         PERM_VIEW_SUBMISSIONS: (PERM_VIEW_ASSET,),
         PERM_PARTIAL_SUBMISSIONS: (PERM_VIEW_ASSET,),
         PERM_CHANGE_SUBMISSIONS: (PERM_VIEW_SUBMISSIONS,),
+        PERM_DELETE_SUBMISSIONS: (PERM_VIEW_SUBMISSIONS,),
         PERM_VALIDATE_SUBMISSIONS: (PERM_VIEW_SUBMISSIONS,)
     }
 
     CONTRADICTORY_PERMISSIONS = {
-        PERM_PARTIAL_SUBMISSIONS: (PERM_VIEW_SUBMISSIONS, PERM_CHANGE_SUBMISSIONS,
-                                      PERM_VALIDATE_SUBMISSIONS),
+        PERM_PARTIAL_SUBMISSIONS: (PERM_VIEW_SUBMISSIONS,
+                                   PERM_CHANGE_SUBMISSIONS,
+                                   PERM_VALIDATE_SUBMISSIONS),
         PERM_VIEW_SUBMISSIONS: (PERM_PARTIAL_SUBMISSIONS,),
         PERM_CHANGE_SUBMISSIONS: (PERM_PARTIAL_SUBMISSIONS,),
+        PERM_DELETE_SUBMISSIONS: (PERM_PARTIAL_SUBMISSIONS,),
         PERM_VALIDATE_SUBMISSIONS: (PERM_PARTIAL_SUBMISSIONS,)
     }
 
     # Some permissions must be copied to KC
-    KC_PERMISSIONS_MAP = {  # keys are KC's codenames, values are KPI's
+    KC_PERMISSIONS_MAP = {  # keys are KPI's codenames, values are KC's
         PERM_CHANGE_SUBMISSIONS: 'change_xform',  # "Can Edit" in KC UI
         PERM_VIEW_SUBMISSIONS: 'view_xform',  # "Can View" in KC UI
         PERM_ADD_SUBMISSIONS: 'report_xform',  # "Can submit to" in KC UI
-        PERM_VALIDATE_SUBMISSIONS: 'validate_xform',  # "Can Validate" in KC UI
+        PERM_DELETE_SUBMISSIONS: 'delete_data_xform',  # "Can delete submissions" in KC UI
+        PERM_VALIDATE_SUBMISSIONS: 'validate_xform',  # "Can validate submissions" in KC UI
```

Comment:
I think "Can Validate" was correct, and the delete permission is called "Can Delete Data" in the KoBoCAT UI:
https://github.com/kobotoolbox/kobocat/blob/02a331a15da3ee2ff00a697a19e54fd2587a51ff/onadata/apps/main/views.py#L391-L395